lib/logstash/inputs/udp.rb

```diff
@@ -67,6 +69,8 @@ def udp_listener(output_queue)
     @udp.bind(@host, @port)
 
     @input_to_worker = SizedQueue.new(@queue_size)
+    metric.gauge(:queue_size, @queue_size)
+    metric.gauge(:threads, @workers)
```

Comment:
I think `workers` would be a better name than `threads`, it reflects the name of the setting.